time_execution/backends/threaded.py

```diff
@@ -74,12 +75,17 @@ def send_metrics():
                 logger.warning('%r write failure %r', self.backend, exc)
 
         while self.has_work():
+            if self.thread:
```

Comment:
We're pretty sure this package causes the segfaults in the monolith (especially i.c.w short lived commands). While the interpreter is shutting down there's apparently still a possibility for the daemon to be executed (which we do not want at that point). I still need to verify this on my local machine if this solves the issue. also see: https://joeshaw.org/python-daemon-threads-considered-harmful/